src/python/pants/engine/streaming_workunit_handler.py

```diff
@@ -26,7 +26,7 @@
 
 @dataclass(frozen=True)
 class TargetInfo:
-    filename: str
+    name: str
```

Comment:
This is a bit of a misnomer. `target.name` really means the `:tgt` portion. For a file target, that would be `path/to/f.ext:original_tgt`.

It's verbose, but maybe `spec_or_filename`? The reality is that this represents both BUILD targets and file targets, which are not the same thing.

Reply:
this is too verbose. name is generic enough and eventually we will add target type and the value represents a target name.
I think naming fields like that is confusing.